hauser/resources.go

```diff
@@ -0,0 +1,51 @@
+package hauser
```

Comment:
can you change this file to `core/core.go` and change the package to `core`?

Reply:
done.